src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -34,7 +34,7 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
      ucs_offsetof(uct_cuda_copy_md_config_t, alloc_whole_reg),
      UCS_CONFIG_TYPE_ON_OFF_AUTO},
 
-    {"MAX_REG_RATIO", "0.1",
+    {"MAX_REG_RATIO", "1.0",
```

Comment:
can't this be too expensive to register much more that the actual communication buffer?